app/embeds/ClickUp.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+import * as React from "react";
+import Frame from "./components/Frame";
+
+const URL_REGEX = new RegExp(
+  "^https://share.clickup.com/b/h/(.*)/(.*)$"
```

Comment:
Did you find any documentation on their share/embeds? It would be good to know what the `b/h/` are here and if it's possible that they will be other values.

Reply:
https://docs.clickup.com/en/articles/3714856-public-sharing

Yes it's possible has others value. I'm chageing it right know.